README.md

```diff
@@ -250,6 +251,11 @@ and you should see:
 ```
 * Selenium requires **Java 8** to run
 
+### Visual Regression Testing
+This is the first iteration of visual regression testing. It is useful to detect side effects or scope of visual changes.
+
+VRT works by gathering the links for the site using the sitemap, then using Nightwatch to navigate throughout the site, capturing an image of each page that will either be used as the baseline for future comparisons or compared to the baseline. The developer must first create the baseline image set for comparisons (sometimes called the golden set), then after making their changes, run an additional task to execute the comparison.
+
```

Comment:
Maybe add a note about it not delving deep into apps, so just because our VRT didn't catch a change doesn't mean there isn't one nested in some specific UI state. User discretion advised.

Reply:
Good idea, will add that in a sec. Thanks man!